autoload/neomake/makers/ft/slim.vim

```diff
@@ -0,0 +1,13 @@
+" vim: ts=4 sw=4 et
+
+function! neomake#makers#ft#slim#EnabledMakers()
+    return ['slimlint']
+endfunction
+
+function! neomake#makers#ft#slim#slimlint()
+    return {
+        \ 'exe': 'slim-lint',
+        \ 'args': ['--no-color'],
+        \ 'errorformat': '%f:%l %m'
```

Comment:
Thank you for the review. I update the errorformat to display correct type now.
now it's '%f:%l [%t] %m'
There are two types of error[E]/warning[W].
example output:
app/views/activities/index.slim:3 [W] TrailingWhitespace: Line contains trailing whitespace
app/views/activities/index.slim:6 [E] Malformed indentation